docs/docs/FAQ.mdx

```diff
@@ -63,8 +63,9 @@ We recommend using a real SSL certificate from a free provider, for example [Let
 
 ### Does Immich change the file?
 
-No, Immich does not touch the original file under any circumstances,  
+No, Immich does not touch the original file under any circumstances,
 all edited metadata are saved in the companion sidecar file and the database.
```

Comment:
Nit: this is a comma splice. A colon, semicolon or two separate sentences would be better.

"modify" is possibly a better word than "touch" here since the file can be deleted.

I also think "metadata is" reads more naturally, though "metadata are" isn't wrong either.